dumb.rb

```diff
@@ -6,3 +6,6 @@ def bad_spacing
 
 dumb = Dumb.new
 p dumb.bad_spacing
+
+
+  message = "this line has extra space everywhere"  
```

Comment:
Useless assignment to variable - message.<br>Inconsistent indentation detected.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.<br>Trailing whitespace detected.